### PR TITLE
522 proposal add active flag to shipping method

### LINKED ISSUE #522: Add active flag to `commercetools_shipping_method ` resource
It would be great if you could add the `active` flag to the `commercetools_shipping_method ` resource.

See https://docs.commercetools.com/api/releases/2024-07-12-added-active-field-to-shipping-methods

### PR DESCRIPTION
**Pull request by gasttor**
Commercetools enhanced the ShippingMethod with an active flag. This PR introduces this attribute to the appropriate resource within the terraform provider.

Commercetools API: https://docs.commercetools.com/api/projects/shippingMethods#shippingmethod

Fixes #522 

### NEW FEATURES | UPGRADE NOTES | ENHANCEMENTS | BUG FIXES | EXPERIMENTS
I added the attribute **ShippingMethod.active**  to the resource *shipping_method*, so that it can be set via terraform.

### CODE REVIEW

File: .changes/unreleased/Added-20241202-214322.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+kind: Added
```
Comment (demeyerthom) [inline .changes/unreleased/Added-20241202-214322.yaml:1]: Can you add a `body` field with the description here? It helps us generate a changelog

**Review by demeyerthom — CHANGES_REQUESTED** [review]
@gasttor looks good! Added one tiny comment about the changelog

File: .changes/unreleased/Added-20241202-214322.yaml (hunk 1 above)
Comment (gasttor) [reply .changes/unreleased/Added-20241202-214322.yaml:1]: Thanks for the review. body-field is added to the yaml-file

**Review by demeyerthom — APPROVED** [review]

**Comment by gasttor** [comment]
Hi @demeyerthom,
I just was made aware that I didn't adapt the documentation for the shipping-method. Will you do it, or shall I provide another PR for that?

File: docs/resources/shipping_method.md

**Pull request merged**